### ISSUE TITLE
bug: False negative accessibility check for some tooltips

### ISSUE THREAD

**Issue by giorgio93p** [saadeghi/daisyui#4740]
### Reproduction URL (Required)

https://play.tailwindcss.com/7u950dTJZk

### What version of daisyUI are you using?

latest

### Which browsers are you seeing the problem on?

Firefox

### Describe your issue

I have created two similar tooltips that differ in the background color of the elements hosting them.
```
      <div class="btn btn-primary tooltip tooltip-primary" data-tip="Hello">
          Hello
      </div>
      <div class="tooltip text-primary tooltip-primary" data-tip="there">
          there
      </div>
```

When checking accessibility via Firefox dev tools, the second tooltip (`data-tip="there"`) is mistakenly marked as having low contrast, because its text is compared to the background of the `div` instead of the background of the tooltip itself. The other tooltip passes the test successfully.

**Comment by saadeghi** [comment]
The goal should be making accessible UI, not to satisfy the flawed measurement tools.  
Even the color contrast measurement these tools are using is a flawed, outdated WCAG from a decade ago, which is not accurate for modern displays.  
But if you want to avoid this false warning, you can use tooltip-content, which is a separate div for the tooltip instead of pseudo elements:  
https://daisyui.com/components/tooltip/#tooltip-with-tooltip-content

Let me know if you have any questions.
